resources/core/charts/apiserver-proxy/templates/job.yaml

```diff
@@ -63,8 +63,20 @@ spec:
             - /bin/bash
             - -c
             - |
-              set -e
-              kubectl -n kyma-system annotate service {{ template "name" . }}-ssl dns.gardener.cloud/class='garden' dns.gardener.cloud/dnsnames='apiserver.'{{ trimPrefix "*." .Values.global.domainName }}'' --overwrite
+              set +e
+              retry=0
+
+              while [[ ${retry} -lt 5 ]]; do
+                result=$(kubectl -n kyma-system annotate service {{ template "name" . }}-ssl dns.gardener.cloud/class='garden' dns.gardener.cloud/dnsnames='apiserver.'{{ trimPrefix "*." .Values.global.domainName }}'' --overwrite)
+                if [[ $? -eq 0 ]]; then
```

Comment:
Doing it like this is ok but tricky. If you insert any other command between `result` and `if` it will stop working. Consider changing it to:
```
cmd=$(...)
result=$?
```